opendbc/car/body/values.py

```diff
@@ -7,7 +7,6 @@
 
 SPEED_FROM_RPM = 0.008587
 
-
 class CarControllerParams:
```

Comment:
two spaces before/after class is the standard Python stylization, and functional PRs should mostly not touch style for diff reasons